schemas/spdx-schema.json

```diff
@@ -63,7 +63,7 @@
           "type" : "string"
         }
       },
-      "required" : [ "created" ],
+      "required" : [ "created", "creator" ],
```

Comment:
I think this does not work, the property is called `creators` (plural) and that should be reflected in the list of `"required"` list

Reply:
I just updated to resolve this issue

Reply:
👍🏼